src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  transition-duration: $transition-duration;
```

Comment:
The `transition-duration` should be set to `300ms` as per the task requirements. Ensure that the variable `$transition-duration` is correctly set to `300ms` in your `variables.scss` file.